### ISSUE TITLE
Can't generate files

### ISSUE THREAD

**Issue by lucasbesen** [sashman/absinthe_gen#1]
Hello.

I'm trying to generate some files but there is something wrong (with import maybe)

```
** (File.Error) could not read file "/Users/lucasbesen/Documents/my-project/deps/absinthe_gen/lib/../templates/types.eex": no such file or directory
    (elixir) lib/file.ex:353: File.read!/1
    lib/eex.ex:184: EEx.compile_file/2
    lib/eex.ex:220: EEx.eval_file/3
    lib/mix/absinthe.gen.scaffold.ex:25: Mix.Tasks.Absinthe.Gen.Scaffold.run/1
    (mix) lib/mix/task.ex:331: Mix.Task.run_task/3
    (mix) lib/mix/cli.ex:79: Mix.CLI.run_task/2
```

**Comment by sashman** [comment]
@lucasbesen thanks very much for your feedback, I'll check this out.

**Comment by sashman** [comment]
Please try version `0.1.1`

**Comment by lucasbesen** [comment]
@sashman still occurring

```
** (File.Error) could not read file "/Users/lucasbesen/Documents/my-project/deps/absinthe_gen/lib/../templates/types.eex": no such file or directory
    (elixir) lib/file.ex:353: File.read!/1
    lib/eex.ex:184: EEx.compile_file/2
    lib/eex.ex:220: EEx.eval_file/3
    lib/mix/absinthe.gen.scaffold.ex:25: Mix.Tasks.Absinthe.Gen.Scaffold.run/1
    (mix) lib/mix/task.ex:331: Mix.Task.run_task/3
    (mix) lib/mix/cli.ex:79: Mix.CLI.run_task/2
```

**Comment by sashman** [comment]
Could you please paste the output of:

```
grep "absinthe_gen" mix.lock
```

**Comment by lucasbesen** [comment]
@sashman sure

```
"absinthe_gen": {:hex, :absinthe_gen, "0.1.1", "ede90a8144adb90145c45da38049fc3a537600394515309b0b0f7b413ba71025", [:mix], [], "hexpm"}
```

**Comment by sashman** [comment]
That's strange, the fix in 0.1.1 should help here: https://github.com/sashman/absinthe_gen/commit/69b4c32cc48282b5c9fbeee5b970045b65091cd1

Can you try using `{:absinthe_gen, github: "sashman/absinthe_gen"}` in the mix file?

**Comment by lucasbesen** [comment]
Now it works.

Thanks @sashman
